### PR TITLE
Font size changing in editor help section

### LINKED ISSUE #35269: Text in the help section of the editor doesn't scale with font settings
**Godot version:**
3.2.rc1

**OS/device including version:**
Linux / Pop_OS

**Issue description:**
The text in the help section of the editor stays the same size no matter if `main_font_size` is changed in the editor settings.
help_font
help_font2

**Steps to reproduce:**
Open a new project, navigate to the help section and alter `main_font_size` in the editor settings.

**Minimal reproduction project:**

### PR DESCRIPTION
**Pull request by siomnnn**
Made font size changing work in the editor's help section, which was previously not affected by `main_font_size`. Fixes #35269 .

### CODE REVIEW

**Comment by akien-mga** [comment]
Well...

As you can see from the code you're removing, there's actually a dedicated option for setting the help font sizes. With this change you're removing its use, thus making the still existing options inactive.

**Comment by noname1477** [comment]
@johnnygossdev suggested that I should not introduce a new option to the settings, but rather scale it with `main_font_size`. Previously, you couldn't change the scale at all (without doing it from code).

**Comment by noname1477** [comment]
Given that we currently have 6 `Font Size` settings (1 basic font, 1 for the code editor, 3 for the help section, 1 for the output) in 3 different places, I would prefer grouping everything font related as a new `Font` tab. Currently, it's just confusing.

**Comment by Calinou** [comment]
We can probably remove all the help font size settings and multiply the main font size instead.

**Comment by johnnygossdev** [comment]
Ah I see the different fonts size options. I should have done a search - I just expected the fonts to be in the same place :sweat_smile:

**Comment by noname1477** [comment]
As suggested by @johnnygossdev and @Calinou I simply removed the useless settings.

**Comment by Calinou** [comment]
Some files were moved to the wrong directories, `editor_settings.cpp` should be in `editor/`, not in the root directory (same for some other files).

**Comment by noname1477** [comment]
Oh, sorry, I didn't see that. I'm still pretty new to GitHub, so I do this mistake all the time.

File: editor/editor_fonts.cpp [hunk 1]
```diff
@@ -243,10 +243,10 @@ void editor_register_fonts(Ref<Theme> p_theme) {
 	p_theme->set_font("title", "EditorFonts", df_title);
 
 	// Documentation fonts
-	MAKE_DEFAULT_FONT(df_doc, int(EDITOR_GET("text_editor/help/help_font_size")) * EDSCALE);
-	MAKE_BOLD_FONT(df_doc_bold, int(EDITOR_GET("text_editor/help/help_font_size")) * EDSCALE);
-	MAKE_BOLD_FONT(df_doc_title, int(EDITOR_GET("text_editor/help/help_title_font_size")) * EDSCALE);
-	MAKE_SOURCE_FONT(df_doc_code, int(EDITOR_GET("text_editor/help/help_source_font_size")) * EDSCALE);
+	MAKE_DEFAULT_FONT(df_doc, (default_font_size + 1) * EDSCALE);
+	MAKE_BOLD_FONT(df_doc_bold, (default_font_size + 1) * EDSCALE);
+	MAKE_BOLD_FONT(df_doc_title, (2 * default_font_size - 1) * EDSCALE);
```
Comment (Calinou) [inline editor/editor_fonts.cpp]: Won't this set the title font size to 27 where it was previously 23? 27 may be a bit too large.

Other than that, this PR looks good to me now.

**Comment by YuriSizov** [comment]
Superseded by https://github.com/godotengine/godot/pull/73106.

**Pull request closed without merging**